src/main/java/org/apache/ibatis/migration/operations/UpOperation.java

```diff
@@ -104,7 +105,7 @@ public UpOperation operate(Connection connection, MigrationLoader migrationsLoad
             scriptReader = migrationsLoader.getScriptReader(change);
 
             long start = System.currentTimeMillis();
-            runner.runScript(scriptReader);
+            runner.runScript(new TemplateReader(migrationsLoader.getScriptReader(change), hookBindings));
```

Comment:
`new TemplateReader(scriptReader, hookBindings)`, since you already called getScriptReader a few lines ago.

Reply:
Also done